catalog/source/index.rst

```diff
@@ -24,3 +24,10 @@ Indices and tables
 * :ref:`modindex`
 * :ref:`search`
 
+
+Terms and Licenses 
+==================
+
+The Data Observatory is a collection of various sources of data with varying licenses. We have worked hard to find you data that will work for the broadest set of use-cases. Please still review the below terms for any dataset you use and please respect the rights of the owners of each dataset. Please use the following link to review third-party data sources that are used in the Data Observatory, along with links to the terms governing the use of each.
+
+http://docs.cartodb.com/cartodb-platform/dataobservatory/license/
```

Comment:
@andrewxhill Could you make this an RST external link a la

    External hyperlinks, like `Python <http://www.python.org/>`_.

From (http://docutils.sourceforge.net/docs/user/rst/quickref.html#external-hyperlink-targets)